data/human/free worlds 2 middle.txt

```diff
@@ -232,6 +232,8 @@ mission "FW Southern Recon 1B"
 			names "republic small"
 			variant
 				"Gunboat (Mark II)"
+			"cargo settings"
+				cargo 0
```

Comment:
How would this alleviate the initial issue? If I understand correctly, the player doesn't have cargo capacity to hold the outfit in their own flagship. Now they've captured the ship with the outfit, but they still can't turn in the quest because they don't have cargo capacity to hold it. Neither does the ship transporting the beam, in this case.

Reply:
This isn't to do with the initial issue, it's for consistency with the other Gunboats that are spawned. 

What this does is clear the cargo hold of the Gunboats, as they spawn with random cargo (e.g. 3 tons of medical, 2 tons of industrial). It doesn't affect the cargo capacity of the Gunboat (Mark II gunboats have 10 cargo space, and this can be increased by getting rid of the outfit expansions they have).

I thought I might lump it in, anyway.